tests/test_e2e.py

```
@@ -74,7 +75,13 @@
                             or (math.isnan(write_res.result_set[i][j])
                                 and math.isnan(read_res.result_set[i][j])))
         return write_res
-
+    
+def query_exception(query: str, message: str, params=None):
+    try:
+        g.query(query, params)
+        assert False, "Expected an error"
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)